tests/src/system/basic/WskBasicNodeTests.scala

```diff
@@ -62,10 +62,27 @@ class WskBasicNodeTests
             }
     }
 
+    it should "Ensure that zipped actions are encoded and uploaded as NodeJS actions" in withAssetCleaner(wskprops) {
```

Comment:
Perhaps we could add a negative test that ensure the proper error message is displayed if a file with a .zip extension is passed without the --kind flag.